File: Razor/Models/Fiche.cs

namespace Models;

using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;


[Table("fiche")]
public class Fiche {
    /// Field
    [Key]
    [Column("id_fiche")]
    public int IdFiche { get; set; }
    
    [Column("name")]
    public string Name { get; set; }
    
    [Column("firstname")]
    public string Firstname { get; set; }
    
    [Column("address")]
    public string Address { get; set; }
    
    [Column("mail")]
    public string Mail { get; set; }
    
    [Column("etat")]
    public int Etat { get; set; }
    
    [Column("salaire")]
    public double Salaire { get; set; }
    
    [Column("admin")]
    public bool Admin { get; set; }
    
    
    
    /// Constructor
    public Fiche(){}
    
    
    /// Getter and setter
    
}

File: Thymeleaf/com/bezkoder/spring/thymeleaf/entity/Fiche.java

package com.bezkoder.spring.thymeleaf.entity;

import jakarta.persistence.*;


@Entity
@Table(name = "fiche")
public class Fiche {
    /// Field
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id_fiche")
    Integer idFiche;
    
    @Column(name = "name")
    String name;
    
    @Column(name = "firstname")
    String firstname;
    
    @Column(name = "address")
    String address;
    
    @Column(name = "mail")
    String mail;
    
    @Column(name = "etat")
    Integer etat;
    
    @Column(name = "salaire")
    Double salaire;
    
    @Column(name = "admin")
    Boolean admin;
    
    
    
    /// Constructor
    public Fiche(){}
    
    
    /// Getter and setter
    public Integer getIdFiche() {
        return idFiche;
    }
    
    public void setIdFiche(Integer idFiche) {
        this.idFiche = idFiche;
    }
    
    public String getName() {
        return name;
    }
    
    public void setName(String name) {
        this.name = name;
    }
    
    public String getFirstname() {
        return firstname;
    }
    
    public void setFirstname(String firstname) {
        this.firstname = firstname;
    }
    
    public String getAddress() {
        return address;
    }
    
    public void setAddress(String address) {
        this.address = address;
    }
    
    public String getMail() {
        return mail;
    }
    
    public void setMail(String mail) {
        this.mail = mail;
    }
    
    public Integer getEtat() {
        return etat;
    }
    
    public void setEtat(Integer etat) {
        this.etat = etat;
    }
    
    public Double getSalaire() {
        return salaire;
    }
    
    public void setSalaire(Double salaire) {
        this.salaire = salaire;
    }
    
    public Boolean getAdmin() {
        return admin;
    }
    
    public void setAdmin(Boolean admin) {
        this.admin = admin;
    }
    
    
}
